src/Helpers/Configuration.ts

```diff
@@ -0,0 +1,6 @@
+import 'Extensions/Array'
+
+export const keysToScreamingSnakeCase = (object: object = {}): {} =>
+  Object.entries(object)
+    .map(([key]) => [key.replace(/([A-Z])/g, capital => '_' + capital).toUpperCase(), key])
```

Comment:
It think it might be good to separate out a function that does this camel case to screaming snake case.

Reply:
Then we could compose together the function something like this

```js
import { _keys, fromPairs, map, pipe } from 'ramda'

import { camelCaseToScreamingSnakeCase } from 'Helpers/camelCaseToScreamingSnakeCase'

const toPair = (s) => [camelCaseToScreamingSnakeCase(s), s]

export const keysToScreamingSnakeCase = pipe(_keys, map(toPair), fromPairs)
```

Reply:
Also, I think there is probably a better name for this function, `keysToScreamingSnakeCase` sounds like we are converting the keys and the values remain the same previous values. What we are really doing is creating a map from ScreamingSnakeCase keys to CamelCase keys. There isn't really a great name that I can think of, but maybe something like `createKeyMapScreamingSnakeCaseToCamelCase`